skimage/feature/corner.py

```diff
@@ -586,7 +586,7 @@ def corner_shi_tomasi(image, sigma=1):
     return response
 
 
-def corner_foerstner(image, sigma=1):
+def corner_foerstner(image, accuracy_thresh, roundness_thresh, sigma=1):
```

Comment:
I would put some default values for these two variables. How's the example values (`0.5`, `0.3`) sound?

Reply:
Also, it would make more sense globally to have `image, sigma, accuracy_thresh, roundness_thresh`, since it'd maintain the model of the other functions. Then, we wouldn't need to deprecate the API. Is that right @jni?

Reply:
> is that right @jni?

The arguments should indeed come after *and* be made keyword-only by using `sigma=1, *, accuracy_thresh=...`, but the API still needs to go through a deprecation cycle somehow, because the return value is being altered. I don't know what the best way to do this is...